doc/source/releases/0.13.0.rst

```diff
@@ -0,0 +1,141 @@
+========================
+Neo 0.13.0 release notes
+========================
+
+??? February 2024
+
+
+This release of Neo sees a lot of changes and improvements, with a documentation rewrite,
+final tweaks and refinements to the object model (in anticipation of a 1.0 release in the next few months),
+a new approach to filtering objects of interest from complex datasets,
+and a large number of bug fixes and performance improvements in IO modules.
+23 people contributed to this release, which is a new record for Neo!
+
+See all `pull requests`_ included in this release and the `list of closed issues`_.
+
+
+Refinements to the Neo object model
+-----------------------------------
+
+In order to simplify the structure of Neo datasets, and make a clearer distinction between the Block-Segment-data
+hierarchy and the more flexible data-grouping functionality, we no longer allow Groups to contain Segments.
+
+We have made the handling of RegionOfInterest subclasses consistent with :class:`ChannelView`,
+i.e., just as a :class:`ChannelView` gives a view of a subset of the channels in a multi-channel :class:`AnalogSignal`,
+so the :class:`RegionOfInterest`` subclasses give views of a subset of the pixels in an :class:`ImageSequence`.
+
+Neo now has automatic handling of relationships between objects.
+In previous versions, the child-parent relationships between objects had to be handled manually,
+for example when you added a :class:`SpikeTrain` to a :class:`Segment`, this created a parent-child relationship,
+but the reverse child-parent relationship was not automatically created, leading to potential inconsistencies.
+Now these relationships are created automatically, as the various child lists
+(``Segment.spiketrains``, ``Segment.analogsignals``, ``Block.segments``, etc.)
+are now represented by pseudo-list objects which take care of the book-keeping behind the scenes.
+
+As a convenience, all Neo containers (:class:`Block`, :class:`Segment`, :class:`Group`)
+now have an :meth:`add()` method that automatically puts the object in the correct list, for example::
```

Comment:
```suggestion
now have an :meth:`add()` method that automatically puts the object in the correct list, for example:
```
Unless the double :: is semantic.

Reply:
it's restructuredtext syntax, it signals that a code block follows